services/integrationtest.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+USAGE="$0 <test-executable> [args...]"
+EXE=${1:?$USAGE}
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+PARENT_PID=$$
+EXIT_STATUS_LOCATION=/dev/shm/integration_test_rc
```

Comment:
Can we store this somewhere, maybe sth in a tmp dir?

Reply:
latest commit uses `mktemp` if that's more compatible on OSX.